cps/templates/user_edit.html

```diff
@@ -40,45 +40,48 @@
             {% endfor %}
         </select>
     </div>
-      <div class="col-sm-6">
```

Comment:
FYI, the large number of changes here is just re-indenting the HTML to standardize it.